updates (#34710)
dotnet/docs: docs/azure/create-azure-account.md

You can create an [Azure account for free](https://azure.microsoft.com/free/dotnet/) and receive 12 months of popular services for free and a $200 credit to explore Azure for 30 days.

## Option 3: Sign up for a pay-as-you-go account

> You can also create a [pay-as-you-go Azure account](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go/). This option includes monthly amounts of select services for free, and charges you for what you use beyond the free limits. There is no upfront commitment and you can cancel at any time.

Reviewer: ```suggestion
You can also create a [pay-as-you-go Azure account](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go/). This option includes monthly amounts of select services for free, and charges you for what you use beyond the free limits. There is no upfront commitment and you can cancel anytime.
```
